packages/flutter/test/gestures/scale_test.dart

```diff
@@ -572,6 +572,34 @@ void main() {
     tap.dispose();
   });
 
+  // Regressing test for https://github.com/flutter/flutter/issues/78941
+  testGesture('First rotation test', (GestureTester tester) {
+    final ScaleGestureRecognizer scale = ScaleGestureRecognizer();
+
+    double? updatedRotation;
+    scale.onUpdate = (ScaleUpdateDetails details) {
+      updatedRotation = details.rotation;
+    };
+
+    final TestPointer pointer1 = TestPointer(1);
+    final PointerDownEvent down = pointer1.down(Offset.zero);
+    scale.addPointer(down);
+    tester.closeArena(1);
+    tester.route(down);
```

Comment:
Why this route? Isn't a `tester.async.flushMicrotasks();` enough here?

Reply:
We need to dispatch the pointer down event to the `ScaleGestureRecognizer` by `route` and the `tester.async.flushMicrotasks();` does not dispatch the event.